menu/aws.go

```diff
@@ -164,6 +195,19 @@ func (m *AWSMenu) Handler() error {
 	}
 }
 
+func (m *AWSMenu) readRegion() (string, error) {
+	region, err := interaction.ReadValue("Region: ")
+	if err != nil {
+		return "", err
+	}
+
+	if !sdkRegions()[region] {
```

Comment:
I think this should be an error rather than just a warning to the user.

Reply:
This is actually intentional to allow Vaulted to work with a new region before the SDK is updated to support it.

But to help ensure that the user doesn't miss the message, line [245](https://github.com/miquella/vaulted/pull/149/files/abbe71178ff751a5e252e2cb62c963659137d9bf#diff-65c99fdf6b74bed7314728ebece8a0cdR245) ensures that each time the vault contents are rendered, a message indicating `(unrecognized region)` is displayed.